src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/scanner/StreamReader.kt

```diff
@@ -133,25 +127,28 @@ class StreamReader(
     }
 
     /**
-     * Peek the next code point (look without moving the pointer)
+     * Moves the pointer forward while [predicate] returns `true`.
```

Comment:
I think this change is wrong, this function still seems to not modify the pointer.